webui/react/src/e2e/utils/envVars.ts

```diff
@@ -0,0 +1,15 @@
+export function webServerUrl(): string {
+  const serverAddess = process.env.PW_SERVER_ADDRESS;
+  if (serverAddess === undefined) {
+    throw new Error(`Expected PW_SERVER_ADDRESS to be set. ${JSON.stringify(process.env)}`);
+  }
+  return serverAddess;
+}
+
+export function apiUrl(): string {
+  return process.env.PW_BACKEND_SERVER_ADDRESS ?? webServerUrl();
```

Comment:
@djanicekpach @johnkim-det can you help me reconsider this environment variable name? maybe like this?
```
PW_BASE_URL="http://localhost:3000/"
PW_SERVER_ADDRESS="http://latest-main.determined.ai:8080/"
PW_USER_NAME="..."
PW_PASSWORD="..."
PW_DET_PATH="/Users/jcomins/miniconda3/envs/det/bin/det"
```
where `PW_SERVER_ADDRESS` would be optional, so this would also work
```
PW_BASE_URL="http://localhost:3000"
PW_USER_NAME="admin"
PW_PASSWORD=""
PW_DET_PATH="/Users/jcomins/miniconda3/envs/det/bin/det"
```
do these names look good to you?

Reply:
I saw `PW_DET_MASTER` in code and I thought that name was good. Is that included?
I don't love `PW_BASE_URL`, but it matches playwright's config so I'm ok with it.